src/main/java/com/google/api/codegen/transformer/py/EnumViewModel.java

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.py;
+
+import com.google.api.client.repackaged.com.google.common.base.Preconditions;
```

Comment:
When you are using Eclipse and you type in `Preconditions` and auto-complete it, Eclipse chooses from the available imports for you automatically. Unfortunately, it typically makes a bad choice and gives you vendored types from other dependencies instead of the main type that you want.